RailwayCo/Assets/Scripts/Unity/Tooltip/TooltipManager.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+public class TooltipManager : MonoBehaviour
+{
+    private static TooltipManager _instance;
+
+    [SerializeField] private Tooltip tooltip;
+
+    public void Awake()
```

Comment:
private this and check for serialized tooltip

Reply:
Fixed in 427cc2b